src/components/Canvas.tsx

```diff
@@ -17,25 +17,28 @@ interface CanvasProps {
   data: string;
   selectedBits: BitSelection;
   isGridView: boolean;
+  isDisplay: boolean; // New prop for play/pause functionality
```

Comment:
if there is not a good reason for using isDisplay as variable name for pause functionality we should stick to isAllPaused, it's more direct and understandable.